app/views/posts/show.html.erb

```diff
@@ -1 +1,15 @@
-<h1>User post</h1>
\ No newline at end of file
+<section>
```

Comment:
- [ ] Views are working properly, nice job! However, It would be great to have some navigation links:

Your views:

<img width="460" alt="Screen Shot 2022-07-21 at 1 55 32 PM" src="https://user-images.githubusercontent.com/83717285/180295410-3db3fa99-0ac1-44ff-8459-a6c1ba08f269.png">

<img width="522" alt="Screen Shot 2022-07-21 at 1 59 41 PM" src="https://user-images.githubusercontent.com/83717285/180295499-06b5259c-5aff-4e3c-90e3-8cd24f55d28a.png">

<img width="542" alt="Screen Shot 2022-07-21 at 1 59 58 PM" src="https://user-images.githubusercontent.com/83717285/180295518-0b9e36bd-b7f0-45f8-9dd9-ffbd43a52129.png">

Styling at this point is not the main goal, but it would be great to have something a little bit similar to the wireframes provided, for example, the comments and likes counter could display '0' when there's is no data, or maybe adding the navigation buttons (See All posts) or links to navigate through the views (Name, Post Title).